devimint/src/tests.rs

```diff
@@ -381,6 +385,10 @@ async fn stress_test_fed(dev_fed: &DevFed, clients: Option<&UpgradeClients>) ->
     // cause the upgrade test to fail
     let assert_thresholds = false;
 
+    // running only one iteration greatly improves the total test time while still
+    // testing the same types of database entries
+    let iterations = 1;
```

Comment:
Probably easy to source it from env var and default to 1. @bradleystachurski

Reply:
Capturing our chat out-of-band: let's keep as-is so it's not  easy to accidentally increase the runtime on the upgrade tests. If someone needs to reuse `stress_test_fed` outside of upgrade tests, we can refactor then.